src/js/disability-benefits/containers/StatusPage.jsx

```diff
@@ -40,10 +41,13 @@ class StatusPage extends React.Component {
       const phase = claim.attributes.phase;
       const showDecision = phase === FIRST_GATHERING_EVIDENCE_PHASE
         && !claim.attributes.waiverSubmitted;
+      const showDocsNeeded = !claim.attributes.decisionLetterSent &&
+        claim.attributes.documentsNeeded &&
+        itemsNeedingAttentionFromVet(claim.attributes.eventsTimeline) > 0;
 
       content = (
         <div>
-          {claim.attributes.documentsNeeded && !claim.attributes.decisionLetterSent
+          {showDocsNeeded
             ? <NeedFilesFromYou claimId={claim.id} events={claim.attributes.eventsTimeline}/>
```

Comment:
Yeah, it probably should do that. There's already a test for this should cover this logic, but I'll make sure it's covering the new case, too.